build/build.sh

```diff
@@ -56,7 +56,7 @@ pandoc --verbose \
 DOCKER_EXISTS="$(command -v docker || true)"
 
 # Create PDF output (unless BUILD_PDF environment variable equals "false")
-if [ "${BUILD_PDF:-true}" != "false" ] && [ -z "$DOCKER_EXISTS" ]; then
+if [ "$(BUILD_PDF:-true)" != "false" ] && [ -z "$DOCKER_EXISTS" ]; then
```

Comment:
This should be parameter expansion (`${BUILD_PDF:-true}`) not command substitution (`$(command -v docker)`). Same below for some variables.

Reply:
Ah that distinction was lost on me, and the root cause of all this back and forth incorrect usage!